test/README.md

```diff
@@ -285,8 +301,8 @@ images that use RPMs created from source (not already published releases).
 #### Rebuilding from Sources Easily
 
 If you build new RPMs, you need to re-run several steps (build the
-RPMs, build the local repos, build the images, download the
-images). Use `./bin/rebuild_source_images.sh` to automate all of those
+RPMs, build the local repos, build the images, download the images).
+Use `./bin/dev_rebuild_source_images.sh` to automate all of those
```

Comment:
I'm not sure why this script needed to be renamed?

Reply:
It does not need to be renamed. I will revert this.
The original idea was to note that this script is for dev flows.